doc/manual/writing_a_driver.rst

```diff
@@ -106,6 +113,34 @@ The controller's code would contain something similar to this: ::
         args = _get_args()
         simple_server_loop(Hello(), args.bind, args.port)
 
+The driver's server should be saved in ~/artiq-dev/artiq/artiq/devices/driver_name/server.py.
+
+__init__.py
+-----------
+This file must exist for porper integration with ARTIQ. It can be empty.
+~/artiq-dev/artiq/artiq/devices/driver_name/__init__.py
+
+Device Manager Udev Rules
```

Comment:
Robert, I've added __setup_serial(self, serial_port) 

Please add to the example how to use serial_for_url() as I've not used it before.